ObsLocTAP.tex

```diff
@@ -554,6 +559,28 @@ \subsection{Observation Locator data model }
 removed from the planning table or they could return to the "Planned" status
 for a possible future schedule.
 
+Tracking type (\textbf{tracking\_type}) is an optional field of the obsplan table
+to characterize the tracking foreseen during the observation execution.  
+The default value would be \textbf{RA/Dec}. 
+For this observations the pointing is mantained on the same position on the sky
+during all the observation period. This kind of tracking is usually known as
+siderial tracking for ground based observations. 
+
+Other possible value is \textbf{solar-system-object-tracking}, to be used when
+the pointing of the observatory varies to maintain the selected solar system
+target on the same position of the observatory point of view. In this case, the
```

Comment:
Replace `solar system target ... point of view` by `solar system target or any other moving object within the observatory field of view`

Reply:
OK